gh.js

```diff
@@ -103,6 +103,12 @@ GH.prototype = {
             cb(null, comment);
         });
     }
+,   getRepo: function(data, cb) {
```

Comment:
I noted in `server.js` that the repo id appears in the pull request output - maybe there is a way to get rid of that additional method altogether? (I haven't checked what data is available to `pr/viewer.jsx`)

Reply:
AFAIK, we currently don't have a method in [`gh.js`](https://github.com/w3c/ash-nazg/blob/master/gh.js) to get the PR. So we need a new endpoint for this.
I could replace `getRepo` with `getPR` which might be more useful in the future. WDYT?

Reply:
what I had in mind was when saving the PR in ash-nazg (via store.addPR), we should save in the same document the id of the github repo - that way, when you retrieve the PR data in `viewer.jsx`, the repo id is readily available.

Reply:
Ok, thanks for the clarification. I updated the code so we can rely on the store for the repository ID: https://github.com/w3c/ash-nazg/pull/163/commits/e30a1bb8673931f0d4712f8d471d2f1901c93086

For the PRs that have been already created, we will miss the repoId and in this case, the link will not appear (there'll be an indication in the text though).